cpp/doxygen/main_page.md

```diff
@@ -107,10 +107,38 @@ To improve performance of small IO requests, `.pread()` and `.pwrite()` implemen
 This setting can also be controlled by `defaults::gds_threshold()` and `defaults::gds_threshold_reset()`.
 
 #### Size of the Bounce Buffer (KVIKIO_GDS_THRESHOLD)
-KvikIO might have to use intermediate host buffers (one per thread) when copying between files and device memory. Set the environment variable ``KVIKIO_BOUNCE_BUFFER_SIZE`` to the size (in bytes) of these "bounce" buffers. If not set, the default value is 16777216 (16 MiB).
+KvikIO might have to use intermediate host buffers (one per thread) when copying between files and device memory. Set the environment variable `KVIKIO_BOUNCE_BUFFER_SIZE` to the size (in bytes) of these "bounce" buffers. If not set, the default value is 16777216 (16 MiB).
 
 This setting can also be controlled by `defaults::bounce_buffer_size()` and `defaults::bounce_buffer_size_reset()`.
 
+#### HTTP Retries
+
+The behavior when a remote IO read returns a error can be controlled through the
```

Comment:
I think the rest of this page is using semantic line breaks, not hard-wrapping at a specific width. Could we match? https://sembr.org/